models/lobby.go

```diff
@@ -251,21 +256,32 @@ func (lobby *Lobby) RemoveSpectator(player *Player) *helpers.TPError {
 	return nil
 }
 
-func (lobby *Lobby) IsFull() bool {
+func (lobby *Lobby) GetPlayerNumber() int {
 	count := 0
 	err := db.DB.Table("lobby_slots").Where("lobby_id = ?", lobby.ID).Count(&count).Error
 	if err != nil {
-		return false
+		return 0
 	}
+	return count
+}
+
+func (lobby *Lobby) IsFull() bool {
+	return lobby.GetPlayerNumber() >= 2*int(lobby.Type)
+}
 
-	return count >= 2*typePlayerCount[lobby.Type]
+func (lobby *Lobby) IsSlotFilled(slot int) bool {
```

Comment:
we already have GetPlayerIdBySlot, this function could wrap around that if you really need it